QuizApp/Client/Account/AccountNetworkClient.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+class AccountNetworkClient: AccountNetworkClientProtocol {
+
+    private let baseNetworkClient: BaseNetworkClientProtocol
+
+    init(baseNetworkClient: BaseNetworkClientProtocol) {
+        self.baseNetworkClient = baseNetworkClient
+    }
+
+    func getAccountDetails() async throws -> AccountDetailsNetworkModel {
+        try await baseNetworkClient.get(
+            url: AccountEndpoints.account.path,
+            params: [:],
+            headers: [HeaderField.contentType.key: HeaderValue.defaultContentType.value],
+            responseType: AccountDetailsNetworkModel.self)
```

Comment:
You can remove `responseType` from base client since Swift can automatically resolve.